### DISCUSSION TITLE
Where is the best place to go for documentation?

### DISCUSSION THREAD

**Discussion by Tiberriver256** [appium/appium-dotnet-driver#806] [Q&A]
The wiki has documentation for v2 and v4.
There is a samples folder.

Is that the best to go for docs?

**Comment by Dor-bl** [comment]
Not sure what you mean by version 2. But the wiki section hasn't been updated for a while now. We have an open issue on updating part of it: https://github.com/appium/dotnet-client/issues/776

**Reply by Tiberriver256** [reply]
My version two was being confused about appium v2 here:
https://github.com/appium/dotnet-client/wiki/Migration-to-Appium-2.x

Sorry about that.

**Reply by Dor-bl** [reply]
Currently the best place for v5 examples can be found in the integration tests : https://github.com/appium/dotnet-client/tree/main/test%2Fintegration
